tests/test_simple_range_query.py

```diff
@@ -0,0 +1,66 @@
+import pandas as pd
+from pandas.testing import assert_frame_equal
+
+from fsql.api import read_partitioned_table
+from fsql.query import ColumnComparator, ColumnRange, SimpleRangeQuery
+
+
+def test_simple_range_query1(tmp_path):
+    # this corresponds to c1=[b, d), c2=*, c3=[c, e)
+    ranges = [
+        ColumnRange("c1", "b", "d"),
+        ColumnRange("c2", "", "", ColumnComparator.wld),
+        ColumnRange("c3", "c", "d"),
+    ]
+    # will be omitted due to c3<b
+    left_bad1 = tmp_path / "c1=b" / "c2=a" / "c3=a"
+    # will be omitted due to c1<b
+    left_bad2 = tmp_path / "c1=a" / "c2=a" / "c3=c"
+    # will be included
+    included1 = tmp_path / "c1=c" / "c2=x" / "c3=d"
```

Comment:
This example should fail (but it does not as simple query returns True immediately for the first column range).

Reply:
If you put `c1=b` it will not be read due to `c3=d`

Reply:
hm, the implementation si wrong. The `c1=b` should lead to acceptance as well -- we can't just `continue` in that case, we have to note we are at the left edge and thus should only compare from below

I'll make the docs a bit better -- I think you understand it as "boolean and of individual ranges, whereas it is supposed to be lexicographical".